### PR TITLE
fix: KEYCLOAK_USER and Https port mapping

### LINKED ISSUE #21: missing KEYCLOAK_USER in the env var
We need `KEYCLOAK_USER` and `KEYCLOAK_PASSWORD` so we can create the initial admin user

### LINKED ISSUE #22: ALB port mapping should go to 8443
The HTTPS listener on ALB should forward to container port `8443` instead of `8080`.

### PR DESCRIPTION
**Pull request by pahud**
fix: missing `KEYCLOAK_USER` in the env var. Closes #21
fix: ALB port mapping should go to `8443`. Closes #22

### CODE REVIEW

**Comment by pahud** [comment]
It works as expected. Copy @wchaws 

圖片

**Comment by pahud** [comment]
will auto merge on CI passes

File: src/keycloak.ts [hunk 1]
```diff
@@ -218,16 +255,18 @@ export class ContainerService extends cdk.Construct {
...
         streamPrefix: 'keycloak',
         logGroup,
       }),
     });
-    kc.addPortMappings({
-      containerPort: 8080,
-    });
+    kc.addPortMappings(
+      { containerPort: 8443 },
+      { containerPort: 8080 },
```
Comment (wchaws) [inline src/keycloak.ts:268]: do we need to remove 8080??

Comment (pahud) [reply src/keycloak.ts:268]: Thanks. Let me cut another PR to remove it.

**Pull request merged**